staging/src/k8s.io/apimachinery/pkg/api/resource/fuzz_test.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resource
+
+import (
+	"testing"
+)
+
+func FuzzParseQuantity(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data string) {
+		_, _ = ParseQuantity(data)
```

Comment:
newbie question, what are the expectations here to know this fails, how are errors surfaced?

Reply:
Here we are fuzzing for panics. If any are found, the fuzzer will report them.